tests/jacobi/include/helpers-jacobi.hpp

```diff
@@ -218,19 +215,20 @@ inline void end_measure(struct counters& timer) {
 }
 
 
-struct Vector_f {
-public:
+struct Vector_f
```

Comment:
If you also change the formatting style, it is more difficult to track what you changed...